frontend/src/app/app-routing.module.ts

```diff
@@ -2,10 +2,16 @@ import { RouterModule, Routes } from '@angular/router';
 
 import { NgModule } from '@angular/core';
```

Comment:
Maybe it would be better to move app component into features folder?

Reply:
Fixed